src/components/ChooseCCApp.tsx

```diff
@@ -7,6 +7,7 @@ import { useIPC } from '../ipc'
 export const ChooseCCApp: React.FC = () => {
   const loadClassFile = useIPC(s => s.loadClassFile)
 
+  // TODO: What about the following code? There is still CCAPP mentioned
```

Comment:
What about this? Must the code be updated or at least the text?

Reply:
It probably would make sense to remove the entire electron config for now.

Reply:
Exactly. But let's do it in a separate PR to keep this env PR clean. OK?